rama-tcp/src/client/mod.rs

```diff
@@ -4,8 +4,9 @@
 pub mod service;
 
 mod connect;
+pub mod pool;
 #[doc(inline)]
-pub use connect::{TcpStreamConnector, default_tcp_connect, tcp_connect};
+pub use connect::{default_tcp_connect, tcp_connect, TcpStreamConnector};
```

Comment:
Are you still on an old Rust version or so? Try to get latest as something tells me you are on an older version, as recently this import order changed?